app/static/js/financialAidOverload.js

```diff
@@ -58,3 +58,21 @@ function openApproveDenyModal(status){
     $("#finOverloadModal").modal("show");
   }
 }
+
+function overloadNoteInsert(textareaID, buttonID) {
+  var overloadNotes = $("#" + textareaID).val();
```

Comment:
Is this a single note? or a list of notes? I would change it to overloadNote if it is a single note being grabbed. However, if it is multiple you can ignore this comment.